pom.xml

```diff
@@ -770,6 +771,12 @@
                 <artifactId>azureblob</artifactId>
                 <version>${jclouds.version}</version>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/com.google.code.gson/gson -->
```

Comment:
Add comment for this transitive dependency. Similar to this one
`<!-- Transitive dependency from jclouds. They're using an old version with security vulnerabilities, so we're 
                pinning it to a newer here. -->`